### ISSUE TITLE
Problem with solution

### ISSUE THREAD

**Issue by sebastian-blandon** [coin-or/Cbc#354]
According to solver CBC the answer is "objective -28700", but if I call manually the result (display Total_Profit), the answer is possitive. It is not a big deal, it is clear that the answer should be possitive, but I could not find the error's cause.

The code was run on ampl, running CBC 2.10.3:

reset;
model;

set PROD:= {"T","C","L"};

param time {PROD} >=0;
param profit {PROD} >=0;
param orders {PROD} >=0;
param avail >=0;

var Make{PROD} >=0 integer;

maximize Total_Profit: sum{p in PROD} profit[p] * Make[p];

s.t. Req {p in PROD}: Make[p] >= orders[p];

s.t. Time: sum{p in PROD} time[p] * Make[p] = avail;

data;

param: time profit orders:=
T 		1	200		10
C 		2	500		20
L 		3	700		15;

param avail := 120;

option solver cbc;
solve Total_Profit;
display Total_Profit;
display Make.lb, Make, Make.ub, Make.rc;

Thank you so much for your help.

**Comment by tkralphs** [comment]
The most likely cause is that the instance is being transformed to a minimization instance somewhere in the interface before passing it to Cbc, so that Cbc doesn't know the original instance was maximization and does not adjust the output accordingly. when you say the answer is "objective -28700" according to Cbc, where is this coming from, the output of the solve? Is that the exact negative of the true result?

**Comment by sebastian-blandon** [comment]
Ted.

The "-28700" is coming from the solver info part, but the output of command "display Total_Profit" is "28700", as showed in the next picture.

![Captura de pantalla 2021-01-09 000102](https://user-images.githubusercontent.com/67570216/104083576-ccdcb180-520d-11eb-9091-352966a3b802.png)

Your opinion makes senses for me, I changed the instance into minimize and I got a possitive answer.

![Captura de pantalla 2021-01-09 000527](https://user-images.githubusercontent.com/67570216/104083648-686e2200-520e-11eb-85b6-1abb9fb67ddf.png)

I just tried the cplex and gurobi solvers, the issue wasn't present. As extra information, I'm running CBC 2.10.3 on AMPL IDE 3.6.4.202012062207.

It's not a problem because I can get the right answer with "display Total_Profit".

Sorry for any inconveniences caused, I'm new to open-source solvers. I am impressed with their good results.

Thank you so much for your help and time, have a nice day!
